network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/video_block.html

```diff
@@ -19,7 +19,7 @@
 <!-- TODO replace bootstrap embed-responsive classes with one created by tailwind component -->
 {% block block_content %}
     <div class="tw-my-4">
-        <div class="embed-responsive embed-responsive-16by9 print:tw-hidden" {% if 'player.vimeo.com' in value.url and value.video_width == "full_width" %}style="padding: 56.25% 0 0 0;"{% endif %}>
+        <div class="embed-responsive {% if responsive_ratio == "" %} {{ responsive_ratio }} {% else %} embed-responsive-16by9 {% endif %} print:tw-hidden" {% if 'player.vimeo.com' in value.url and value.video_width == "full_width" %}style="padding: 56.25% 0 0 0;"{% endif %}>
```

Comment:
Need to clear the 16by9 responsive ratio css class otherwise a full width video in accordion would have huge padding above/below the video. See the highlighted areas in the screenshot below

![image](https://user-images.githubusercontent.com/2896608/231599572-c03634ac-e307-42b7-9b74-2d3691edfd19.png)